src/seedu/addressbook/data/person/Person.java

```diff
@@ -84,4 +85,14 @@ public String toString() {
         return getAsTextShowAll();
     }
 
+    /**
+     * Defines comparator used in sorting names by alphabetical order.
+     */
+    public static Comparator<Person> PersonComparator = new Comparator<Person>() {
+        public int compare(Person person1, Person person2) {
+            String PersonName1 = person1.getName().toString().toUpperCase();
+            String PersonName2 = person2.getName().toString().toUpperCase();
+            return PersonName1.compareTo(PersonName2);
+        }
+    };
```

Comment:
Combining class definition and object creation is not a recommended practice. Can you think of a reason why ?